README.md

```diff
@@ -71,4 +71,4 @@ Follow these steps to set up the project locally:
    bin/dev # use to enable tailwind and hot reload
    ```
 
-5. Visit `http://localhost:3000` in your web browser to see the application running.
+5. Visit `http://localhost:3000` in your web browser to see the application running (Google maps won't work otherwise).
```

Comment:
good call, I (almost) always use 3000 so I didn't think about others. 
Could you also please make it: 

> Visit `http://localhost:3000` or `http://0.0.0.0:3000` in your web...

Both are whitelisted and it isn't mentioned anywhere.

Reply:
Done in 4113dad.

While we're at it, do you mind adding `http://127.0.0.1:3000` and `http://[::1]:3000` to the whitelist if possible?

My rationale is that when you start your Rails server, it states:
```
* Listening on http://127.0.0.1:3000
* Listening on http://[::1]:3000
```

So it might be where people go by default (and it's easy to click on it).

Reply:
sounds good! 
I'll do my best to add it today, review everything and hopefully we can merge it then. Thanks again!

Reply:
I've added `http://127.0.0.1:3000` but `http://[::1]:3000/` is not whitelisted by Google for whatever reason - even though I'm adding it in the same way. There is no IPv6 section there so I don't have any other options. We'll have to live without it

Reply:
Partial improvement is better than no improvement :slightly_smiling_face: 

Thanks!